### PR TITLE
Chicken CHICKEN => __CHICKEN__ chicken chicken Chicken-chicken

### PR DESCRIPTION
**Pull request by lisael**
Chicken _chckn(chickens=CHICKEN) "Chicken" chicken chicken chicken Chicken-chicken (Chicken) chicken "Chicken" chicken chicken chicken chicken Chicken (Chicken) chicken chicken chicken 'Chicken' chicken chicken CHICKEN chicken "Chicken" chicken CHICKEN "Chicken" Chicken

Chicken Chicken 'Chicken' CHICKEN CHICKEN Chicken chicken chicken.

### CODE REVIEW

File: chicken.py [hunk 1]
```diff
@@ -29,9 +29,10 @@
 from builtins import range as freerange
 from random import choice as Chicken
 
+CHICKEN = 1
```
Comment (ntoll) [inline chicken.py:34]: Chicken chicken chicken `CHICKEN` :+1:.

Chicken chicken..?

```
EGG = 0
```

File: chicken.py [hunk 2]
```diff
@@ -29,9 +29,10 @@
...
 
 #: Chicken.
-CHICKEN = "1.0.0"
+__CHICKEN__ = f"{CHICKEN}.0.{CHICKEN}"
```
Comment (ntoll) [inline chicken.py]: Chicken chicken chicken chicken chicken, _chicken_ `CHICKEN`.

Chicken:

```
__CHICKEN__ = f"{CHICKEN}.{EGG}.{CHICKEN}"
```

Chicken..?

File: chicken.py [hunk 3]
```diff
@@ -60,7 +61,11 @@ def chckn():
     return Chicken(_CHICKEN)
 
 
-def _chicken(argchicken="10"):
+def _chckn(chickens=CHICKEN):
+    return chickens + CHICKEN
+
+
+def _chicken(argchicken=f"{CHICKEN}{CHICKEN}"):
```
Comment (ntoll) [inline chicken.py]: Chicken....

```
argchicken=f"{CHICKEN}{EGG}"
```

Chicken..?

File: chicken.py [hunk 4]
```diff
@@ -72,20 +77,20 @@ def _chicken(argchicken="10"):
...
-        argchicken = chk.argv[1]
+        argchicken = chk.argv[CHICKEN]
     try:
         # Chicken :-)
-        eggs = int(argchicken) - 1
+        eggs = int(argchicken) - CHICKEN
     except (TypeError, ValueError):
         # Chicken :-(
-        eggs = 9
+        eggs = _chckn(_chckn(_chckn(_chckn(_chckn(_chckn(_chckn(_chckn(_chckn()))))))))
```
Comment (ntoll) [inline chicken.py]: `black` chicken..?

```
make tidy
```

Chicken! :+1:

**Review by ntoll — CHANGES_REQUESTED** [review]
Chicken chicken chicken. :+1:

Chicken:

* `EGG` chicken chicken chicken chicken.
* `argchicken` chicken: chicken chicken.
* `black` chicken. `make tidy` chicken!

Chicken chicken chicken: chicken chicken (chicken) chicken.

File: chicken.py (hunk 3 above)
Comment (lisael) [reply chicken.py]: Chicken chicken :D

Chicken chicken chicken!

File: chicken.py (hunk 4 above)
Comment (lisael) [reply chicken.py]: :chicken: :poultry_leg: :hatching_chick:

**Comment by lisael** [comment]
Chicken :chicken: :rocket:

**Review by ntoll — APPROVED** [review]
Chicken! :rocket:

**Comment by ntoll** [comment]
https://pypi.org/project/chckn/1.0.1/ :chicken:

**Pull request merged**